Dockerfile

```diff
@@ -0,0 +1,14 @@
+# @v0.0.1
+FROM sea5kg/wsjcpp:latest
```

Comment:
As I understand, this is the "parent image" you're building upon ? can we use something more generic / public ?